### PR TITLE
Investigate Angular/Grafana integration

### PR DESCRIPTION
**Pull request by dan-du-car**
# PR Details
Investigate Angular/Grafana integration
## Description
- Add grafana, apache2, telematic message server and NATs docker services to docker-compose.yml
- Update MYSQL database with telematic and grafana tables. Details refer to: https://github.com/usdot-fhwa-OPS/V2X-Hub/tree/prototype_dashboard
- Add UI component to include grafana dashboard in an iframe.

List of dashboards:
image

Detail of a dashboard:
image

## Related Issue
https://usdot-carma.atlassian.net/browse/VH-1348

## Motivation and Context
V2xHub UI prototype

## How Has This Been Tested?
Local integration test

## Types of changes

- [ ] Defect fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that cause existing functionality to change)

## Checklist:

- [ ] I have added any new packages to the sonar-scanner.properties file
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING**](https://github.com/usdot-fhwa-stol/carma-platform/blob/develop/Contributing.md) document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: src/app/services/authentication/authentication.service.ts [hunk 1]
```diff
@@ -36,19 +36,19 @@ export class AuthenticationService {
   }
 
   loginResponse(response: Login): void {
-    console.log("Login Response : " + response);
+    console.log("Login Response : " + JSON.stringify(response));
```
Comment (paulbourelly999) [inline src/app/services/authentication/authentication.service.ts]: Why is this log statement required

File: src/app/components/plugins/plugins.component.html [hunk 2]
```diff
@@ -26,5 +26,8 @@ <h3 class="plugin-filter-item">FILTER PLUGINS</h3>
     <mat-tab label="Events">Events Content</mat-tab>
     <mat-tab label="Messages">Message Content</mat-tab>
     <mat-tab label="Users">User Content</mat-tab>
+    <mat-tab label="Dashboard">
```
Comment (paulbourelly999) [inline src/app/components/plugins/plugins.component.html]: Can we rename this to Grafana or Telematics

**Review by paulbourelly999 — CHANGES_REQUESTED** [review]
Small changes

File: src/app/services/authentication/authentication.service.ts (hunk 1 above)
Comment (dan-du-car) [reply src/app/services/authentication/authentication.service.ts]: It prints the content of the object in JSON string. The current print out show `[object]`?

File: src/app/components/plugins/plugins.component.html (hunk 2 above)
Comment (dan-du-car) [reply src/app/components/plugins/plugins.component.html]: Addressed https://github.com/usdot-fhwa-OPS/v2xhub-angular-ui-prototype/pull/1/commits/861d750f18b82e4210a4eba55055c0a7e7d71d90

File: src/app/services/authentication/authentication.service.ts (hunk 1 above)
Comment (paulbourelly999) [reply src/app/services/authentication/authentication.service.ts]: We know this functionality works. I am just confused why we are keeping this log statement that does little to tell us anything about how the system is running

Comment (dan-du-car) [reply src/app/services/authentication/authentication.service.ts]: Removed https://github.com/usdot-fhwa-OPS/v2xhub-angular-ui-prototype/pull/1/commits/003442118db552098f6bd39ddcda27bf66a91bec

**Review by paulbourelly999 — APPROVED** [review]

**Pull request merged**